clusterctl-settings.json

```diff
@@ -2,6 +2,6 @@
   "name": "infrastructure-metal3",
   "config": {
     "componentsFile": "infrastructure-components.yaml",
-    "nextVersion": "v1.7.99"
+    "nextVersion": "v1.9.99"
```

Comment:
Why are we jumping two versions here? Should we jump just one, or how did the previous work when it kept pointing to 1.7.99?

Reply:
.I think it should be v1.8.99

Reply:
Got it, we are following CAPI

Reply:
it depends on how we set our config in the test folder in this block: https://github.com/metal3-io/cluster-api-provider-metal3/blob/7ad10785badd816e78cd9d7dc8fbd0bbc0b74c72/test/e2e/config/e2e_conf.yaml#L111 , I can change it to 1.8.99 in dev-env and in CAPM3 main branch I can change so on  1.8.99 points to main and it will work fine, but it seems bit wrong to me. Also CAPI and other providers do it like this. ref CAPI: https://github.com/kubernetes-sigs/cluster-api/blob/be86b82e7e30a844bca141ff8bcdc450b0499549/cmd/clusterctl/hack/create-local-repository.py#L74 , CAPV: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/c14a875c30ea8dd66fcdc5b3336620f5d0742201/clusterctl-settings.json#L5C24-L5C26 (CAPV latest release is 1.11)